common/Stress/Azure.Sample.Stress/InvalidConstructorTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Test.Stress;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Sample.Stress
+{
+    // Verifies the perf framework prints a nice error for invalid constructors
+    public class InvalidConstructorTest : StressTest<StressOptions, StressMetrics>
```

Comment:
This seems like it should be part of the Test project and not a sample. Rationale: It is specific to a test-only scenario and not something that we would consider good practice. Having it in a sample could be confusing for those looking to onboard to using the framework.